### DISCUSSION TITLE
Install not working

### DISCUSSION THREAD

**Discussion by anuprathod774** [rustdesk/rustdesk#9872] [General]
I have changed the app name and logos. It was built successfully when I opened it for the first time, and it is working great.

But When I click install from the left pane, it copies all the files and folders from the release folder and still shows install. Please help

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/763, go to [discord](https://discord.gg/nDceKgxnkV) please, please search there first, many guys have asked similar question because you are not the most special though you always think you are. Usually nobody will help you with build or self-hosting here.
